elyra/pipeline/parser.py

```diff
@@ -254,8 +257,14 @@ def _get_remaining_component_params(node: Dict):
                 if key in ["filename", "runtime_image", "cpu", "gpu", "memory", "dependencies",
                            "include_subdirectories", "env_vars", "outputs", "ui_data", "component_source_type"]:
                     continue
-                # Remove parameter id if a unique identifier was added during component properties parsing
+                # Remove unique identifier of parameter id if one was added during component properties parsing
                 if "elyra_outputs_" in key:
                     key = key.replace("elyra_outputs_", "")
+                # For path inputs and outputs, grab the content in order to pass to contructor
+                if "elyra_path_" in key:
```

Comment:
Could this be `key.startswith("elyra_path_")` instead or is "elyra_path_" a substring?  (Same comment for "elyra_outputs_" above).

Reply:
I'll have to think through the scenarios here to see if there's a better way to pass variable type info than in the key like this. But if there isn't, I think I'll change this to check "elyra_path_" first (and use `startswith()` to do so) and do the key and value reassignment. And then afterwards check for "elyra_outputs_" with `startswith()` as well, since they'll always be at the start of the string